tests/app/v2/notifications/test_get_notifications.py

```diff
@@ -413,10 +414,11 @@ def test_get_all_notifications_filter_by_multiple_statuses(client, sample_templa
 
 
 def test_get_all_notifications_filter_by_failed_status(client, sample_template):
+    # TODO had to change temporary-failure, permanent-failure, technical-failure to failed
     created_notification = create_notification(template=sample_template, status="created")
     failed_notifications = [
         create_notification(template=sample_template, status=_status)
-        for _status in ["technical-failure", "temporary-failure", "permanent-failure"]
+        for _status in ["failed", "failed", "failed"]
```

Comment:
This could just be shortened to `['failed',]` and it'll still work as expected!

```suggestion
        for _status in ['failed',]
```